pkg/pointerdb/audit/audit_segment.go

```diff
@@ -0,0 +1,46 @@
+// Copyright (C) 2018 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package audit
+
+import (
+	"context"
+
+	p "storj.io/storj/pkg/paths"
+	pdbclient "storj.io/storj/pkg/pointerdb/pdbclient"
+	meta "storj.io/storj/pkg/storage/meta"
+)
+
+//Randomly choose a pointer from pointerdb.
+//Library will return a random pointer. Has a super
+// user privilege: no one else should be able to randomly
+//choose any random pointer. This is purely for auditing,
+//and auditing has special privileges.
+
+// ProcessPointer to get and process Pointer data
+
+// Get list pagination of  pointers
+// keep track  of last path used so not to duplicate
+// randomly select from   that list
+// process pointter information for segment
+// send that
+
+// ProcessPointer to get randomized pointer
+type ProcessPointer interface {
```

Comment:
Remove this interface, it's not that useful. Might as well use the structs directly.